Sources/TensorFlow/Operators/Basic.swift

```diff
@@ -1361,6 +1364,50 @@ extension Tensor {
         "Axes must have rank 0 or rank 1; axes has rank \(axes.rank) with values \(axes.scalars).",
         file: file,
         line: line)
-    return areAxesInRange(axes.scalars)
+    return isInRange(axes: axes.scalars)
+  }
+
+  /// Ensures the provided axes refer to axis of `self` (are in range `-rank..<rank`).
+  @usableFromInline
+  func ensureValid(
```

Comment:
OK, so we should choose either “valid” or “in range” to denote whether the something refers to a valid axis.  I vote for the former.

Reply:
Renamed all of the above to `{is|are}Valid` to work with `ensureValid`'s below.

Reply:
W00t!